app/components/partner/client-uploads-stat-chart/index.js

```diff
@@ -65,10 +69,12 @@ export default class PartnerClientUploadsStatChartComponent extends Component {
       },
     },
     {
-      key: this.intl.t('monthWithNum', {
+      key: 'month',
+      axisText: this.intl.t('month'),
+      buttonSelectorText: this.intl.t('monthWithNum', {
```

Comment:
can you change this to the new month inlt.

Reply:
Are you referring to the axisText?

Reply:
Done